tools/setup-windows/setup-windows.csproj

```diff
@@ -1,38 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{73DF9E10-E933-4222-B8E1-F4536FFF9FAD}</ProjectGuid>
+    <TargetFrameworks>net472;netcoreapp3.1</TargetFrameworks>
```

Comment:
I don't see why `setup-windows.exe` needs to be `netcoreapp3.1`? Can we just build it for `net472`, since Windows comes with .NET framework preinstalled?

Reply:
It does not need to be `netcoreapp3.1` but dependencies forced me to add it.

```
./external/debugger-libs/UnitTests/MonoDevelop.Debugger.Tests.TestAppCore/MonoDevelop.Debugger.Tests.TestAppCore.csproj:5:    <TargetFramework>netcoreapp3.0</TargetFramework>
./external/xamarin-android-tools/tests/Xamarin.Android.Tools.AndroidSdk-Tests/Xamarin.Android.Tools.AndroidSdk-Tests.csproj:4:    <TargetFrameworks>net461;netcoreapp3.1</TargetFrameworks>
./external/xamarin-android-tools/src/Xamarin.Android.Tools.AndroidSdk/Xamarin.Android.Tools.AndroidSdk.csproj:4:    <TargetFrameworks>netstandard2.0;netcoreapp3.1</TargetFrameworks>
./external/Java.Interop/tools/jcw-gen/jcw-gen.csproj:4:    <TargetFrameworks>net472;netcoreapp3.1</TargetFrameworks>
./external/Java.Interop/tools/param-name-importer/param-name-importer.csproj:3:    <TargetFrameworks>net472;netcoreapp3.1</TargetFrameworks>
./external/Java.Interop/tools/param-name-importer/param-name-importer.csproj:15:    <!-- This package erroneously contains /netcoreapp3.1/SgmlReader.exe and /netcoreapp3.1/SgmlReaderDll.dll.
./external/Java.Interop/tools/generator/generator.csproj:4:    <TargetFrameworks>net472;netcoreapp3.1</TargetFrameworks>
./external/Java.Interop/tools/class-parse/class-parse.csproj:4:    <TargetFrameworks>net472;netcoreapp3.1</TargetFrameworks>
./external/Java.Interop/tools/logcat-parse/logcat-parse.csproj:4:    <TargetFrameworks>net472;netcoreapp3.1</TargetFrameworks>
./external/Java.Interop/external/xamarin-android-tools/tests/Xamarin.Android.Tools.AndroidSdk-Tests/Xamarin.Android.Tools.AndroidSdk-Tests.csproj:4:    <TargetFrameworks>net461;netcoreapp3.1</TargetFrameworks>
./external/Java.Interop/external/xamarin-android-tools/src/Xamarin.Android.Tools.AndroidSdk/Xamarin.Android.Tools.AndroidSdk.csproj:4:    <TargetFrameworks>netstandard2.0;netcoreapp3.1</TargetFrameworks>
./external/Java.Interop/build-tools/jnienv-gen/jnienv-gen.csproj:5:    <TargetFrameworks>net472;netcoreapp3.1</TargetFrameworks>
./external/Java.Interop/src/Java.Interop/Java.Interop.csproj:4:    <TargetFrameworks>netstandard2.0;netcoreapp3.1</TargetFrameworks>
./src/Mono.Android/Mono.Android.csproj:8:    <TargetFrameworks>monoandroid10;netcoreapp3.1</TargetFrameworks>
./src/Mono.Android/Mono.Android.csproj:34:  <PropertyGroup Condition=" '$(TargetFramework)' == 'netcoreapp3.1' ">
./src/Mono.Android/Mono.Android.csproj:85:  <ItemGroup Condition=" '$(TargetFramework)' == 'netcoreapp3.1' ">
./src/Mono.Android/Mono.Android.csproj:362:  <!-- Only build the 'netcoreapp3.1' version of 'Mono.Android.dll' once for the latest stable Android version. -->
./src/Mono.Android/Mono.Android.csproj:363:  <PropertyGroup Condition=" '$(TargetFramework)' == 'netcoreapp3.1' And '$(AndroidFrameworkVersion)' != '$(AndroidLatestStableFrameworkVersion)' ">
./src/Mono.Android.Export/Mono.Android.Export.csproj:8:    <TargetFrameworks>monoandroid10;netcoreapp3.1</TargetFrameworks>
./src/Mono.Android.Export/Mono.Android.Export.csproj:23:  <PropertyGroup Condition=" '$(TargetFramework)' == 'netcoreapp3.1' ">
./src/Mono.Android.Export/Mono.Android.Export.csproj:54:  <ItemGroup Condition=" '$(TargetFramework)' == 'netcoreapp3.1' ">
./src/Mono.Android.Export/Mono.Android.Export.csproj:64:  <!-- Only build the 'netcoreapp3.1' version of 'Mono.Android.Export.dll' once. -->
./src/Mono.Android.Export/Mono.Android.Export.csproj:65:  <PropertyGroup Condition=" '$(TargetFramework)' == 'netcoreapp3.1' And '$(AndroidFrameworkVersion)' != '$(AndroidLatestStableFrameworkVersion)' ">
./src/Microsoft.Android.Sdk.ILLink/Microsoft.Android.Sdk.ILLink.csproj:4:    <TargetFramework>netcoreapp3.0</TargetFramework>
```

Reply:
`setup-windows.csproj` doesn't have any dependencies? `setup-windows.csproj` doesn't reference any other projects? No projects reference `setup-windows.csproj`?

Reply:
JonP checked which projects were already migrated, so I took Java.Interop projects as starting point. And almost all of those projects do target `netcoreapp3.1`. I started with  `./tools/` subfolder and `vswhere.csproj` and `setup-windows.csproj` were one of the first projects migrated. Thus `netcoreapp3.1`.
I have removed it from this and other projects.